timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/database/tinkerpop/TinkerPopOperations.java

```diff
@@ -1352,6 +1359,17 @@ private Vertex createRdfEntity(Vre vre, String rdfUri) {
     return graph.traversal().V().has(ENTITY_TYPE_NAME_PROPERTY_NAME, defaultEntityTypeName(vre)).toStream().findAny();
   }
 
+  private Optional<Vertex> getPredicateValueTypeVertexFor(Vre vre) {
+
+    return graph.traversal().V()
```

Comment:
Also. In my branch I cached these in a hashmap. So I'll have to adapt that code after your branch merges